docs/DeveloperGuide.md

```diff
@@ -378,234 +375,353 @@ Use case ends.
+* **Use Case**: UC09 - Remove "Completed" Marking from Order
+* **Preconditions**: There are orders displayed in the customer's list.
 * **Guarantees**:
     * Specified order will be reverted to "Pending" status if the input parameters are valid.
 
 **MSS**
 
-1. User wants to remove "Completed" Marking from Order due to mistake.
-2. User finds the order they want to mark as completed.
-3. User specifies the index of the order.
-4. SellSavvy updates the status of the order and indicates that the action is successful.
+1. User finds the order they want to mark as completed.
+2. User specifies the index of the order.
+3. SellSavvy updates the status of the order and indicates that the action is successful.
 
 Use case ends.
 
 **Extensions**
 
-* 3a.  The specified order is not marked as “Completed” in the first place.
-    * 3a1. SellSavvy displays a message stating that the order is not marked as completed in the first place.
+* 2a. SellSavvy detects that the order index is missing or non-positive.
+    * 2a1. SellSavvy displays an error about the format and states the command format.
 
   Use case ends.
 
 
-* 3b. There are no orders with the specified index.
-    * 3b1. SellSavvy displays an error that the order index is invalid.
+* 2b.  The specified order is not marked as “Completed” in the first place.
+    * 2b1. SellSavvy displays a message stating that the order is not marked as completed in the first place.
 
   Use case ends.
 
-**Use case 9: Delete an order**
 
-* **System**: SellSavvy
-* **Actor**: User
-* **Use Case**: UC09 - Delete an Order
-* **Preconditions**: A customer's list of orders is being displayed.
+* 2c. There are no orders with the specified index.
+    * 2c1. SellSavvy displays an error that the order index is invalid.
+
+  Use case ends.
+
+**Use case 10: Delete an order**
+
+* **Use Case**: UC10 - Delete an Order
+* **Preconditions**: There are orders displayed in the customer's list.
 * **Guarantees**:
     * An order made by the customer will be deleted if input parameters are valid.
 
 **MSS**
 
-1. User wants to delete an order.
-2. User finds the order they want to delete.
-3. User deletes the order by their index.
-4. SellSavvy updates the displayed list of orders that the action is successful.
+1. User finds the order they want to delete.
+2. User deletes the order by its index.
+3. SellSavvy updates the displayed list of orders that the action is successful.
+
+Use case ends.
+
+**Extensions**
+
+* 2a. SellSavvy detects that the order index is missing or non-positive.
+    * 2a1. SellSavvy displays an error about the format and states the command format.
+
+  Use case ends.
+
+
+* 2b. There are no orders with the specified index.
+    * 2b1. SellSavvy displays an error that the order index is invalid.
+
+  Use case ends.
+
+**Use case 11: Edit an Order's Details**
+
+* **Use Case**: UC11 - Edit a Order's Details
+* **Preconditions**: There are orders displayed in the customer's list.
```

Comment:
same as above

Reply:
Changing to `A customer's list of order's is being displayed.`